include/seastar/http/request.hh

```diff
@@ -313,10 +313,10 @@ struct request {
      */
```

Comment:
What for? Should be in the patch comment

Reply:
Thanks, I added comment in https://github.com/scylladb/seastar/pull/2579.